hipBone.cpp

```diff
@@ -38,6 +38,9 @@ int main(int argc, char **argv){
     if (settings.compareSetting("VERBOSE", "TRUE"))
       settings.report();
 
+    if (settings.compareSetting("VERSION", "TRUE"))
+      std::cout << "hipBone:'" << HIPBONE_VERSION_STR << std::endl;
```

Comment:
This is happening after `MPI_Init` is called, so every rank will print it.  Is that what you expected?  It might be cleaner to simply have only rank 0 print it.

Why is there a superfluous quotation mark?  Can you please do this instead:
```
std::cout << "hipBone " << HIPBONE_VERSION_STR << std::endl;
```